.github/ISSUE_TEMPLATE/hcl-report.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Report Hardware Compatibility List
+about: 'Report information about compatible hardware that you are using in your setup'
+title: ''
+labels: 'P: default, T: hcl'
+assignees: ''
+
+---
+
+**Dasharo version**
+<!--(The version of Dasharo you're using (e.g., `v0.2.0`))-->
+
+**CPU**
+
+* Processor name: <!--(full name including vendor, brand and CPU number))-->
+* Core name: <!--(CPU core codename))-->
+* CPU base speed: <!--(based CPU speed))-->
```

Comment:
All of these can be taken from datasheet. I believe all we need is vendor/model.

Reply:
added `(optional)` notes